src/odemis/driver/test/xt_client_test.py

```diff
@@ -67,14 +67,14 @@
               }
 
 CONFIG_FIB_SCANNER = {"name": "fib-scanner", "role": "ion", "channel": "ion2"}
-CONFIG_FIB_SEM = {"name": "sem", "role": "sem", "address": "PYRO:Microscope@192.168.31.162:4242",
+CONFIG_FIB_SEM = {"name": "sem", "role": "sem", "address": "PYRO:Microscope@0.0.0.0:4242",
```

Comment:
I will remove these changes before merging

Reply:
Normally this is already handled by setting the environment variable TEST_NOHW = "sim", so you shouldn't need to change the source code.